thirdparty_builtin/CMakeLists.txt

```diff
@@ -70,27 +70,36 @@ if(ENABLE_TESTS)
         set(BUILD_GTEST ON  CACHE BOOL "")        
     endif()
 
-
-    # Enable builtin google test 
-    add_subdirectory(googletest-release-1.8.0)
-
-    blt_register_library(NAME gtest
-                         INCLUDES ${gtest_SOURCE_DIR}/include
-                         LIBRARIES gtest_main gtest
-                         DEFINES  ${gtest_defines})
-    
+    message(STATUS "Google Test Support is ${ENABLE_GTEST}")
     message(STATUS "Google Mock Support is ${ENABLE_GMOCK}")
     
-    if(ENABLE_GMOCK)
-        blt_register_library(NAME gmock
-                             INCLUDES ${gmock_SOURCE_DIR}/include
-                             LIBRARIES gmock_main gmock
+    #
+    # Guard of googletest w/ ENABLE_GTEST
+    # In BTL ENABLE_GTEST is also required when using ENABLE_GMOCK
+    #
+    
+    if(ENABLE_GTEST)
+        # Enable builtin google test 
+        add_subdirectory(googletest-release-1.8.0)
+
+        blt_register_library(NAME gtest
+                             INCLUDES ${gtest_SOURCE_DIR}/include
+                             LIBRARIES gtest_main gtest
                              DEFINES  ${gtest_defines})
+         if(ENABLE_GMOCK)
```

Comment:
@davidbeckingsale  I added logic to force ENABLE_GTEST to ON, if ENABLE_GMOCK is ON